plone/app/layout/globals/context.py

```diff
@@ -163,7 +169,14 @@ def folder(self):
 
     @memoize
     def is_folderish(self):
-        return bool(getattr(aq_base(aq_inner(self.context)), 'isPrincipiaFolderish', False))
+        cond1 = bool(
+            getattr(aq_base(aq_inner(self.context)),
+                    'isPrincipiaFolderish', False))
+        cond2 = False
+        if HAS_PAC:
+            if ICollection.providedBy(self.context):
+                cond2 = True
+        return cond1 or cond2
```

Comment:
Yep, but we can call folderContents & co on it.
And we must keep that behavior as many addons, like eea.facetednaviguation relies on it
cc @avoinea 

This bit of hack is i think just sufficient to support new Collections.